sound/soc/sof/topology.c

```diff
@@ -2387,13 +2387,12 @@ static int sof_link_unload(struct snd_soc_component *scomp,
 			continue;
 
 		if (strcmp(link->name, sof_dai->name) == 0)
-			break;
+			goto find;
 	}
 
-	if (!sof_dai) {
-		dev_err(sdev->dev, "failed to find dai %s", link->name);
-		return -EINVAL;
-	}
+	dev_err(sdev->dev, "failed to find dai %s", link->name);
+	return -EINVAL;
+find:
```

Comment:
@bardliao sorry for the nitpick :) but can this be "found" instead?

Reply:
@ranj063 of course :)